Stripe/PublicHeaders/STPAPIResponseDecodable.h

```diff
@@ -11,6 +11,11 @@
 @protocol STPAPIResponseDecodable <NSObject>
 
 /**
+ A string representing the object's type. May be nil.
+ */
++ (nullable NSString *)object;
```

Comment:
That's fair – I'll add a private superprotocol that adds an optional `stripeObject` method.